docs/spec/distribution/triggers.md

```diff
@@ -0,0 +1,31 @@
+# Triggers
```

Comment:
What about slashing events (which modify validator stake and delegation stake, and not necessarily in the same proportion)?

Reply:
Also, what about parameter changes (e.g. enacted by `ParameterChangeProposal`s)?

Reply:
Slashing events is a situation that needs to be considered:

Side note: Equal slashing of delegators and validators is a design constraint of this system - we can talk about it more outside of github if you'd like. 

These slashing events will need to have a new special computation not yet outlined in this spec: can be processed similarly with a new pool as per to unbonding validators. more discussion here: https://github.com/cosmos/cosmos-sdk/issues/1905

________
Parameter changes will not trigger any new changes because parameters relevant to distribution affect the tokens flowing into the distribution system but do not affect the tokens already contained within this distribution system.